.github/workflows/feature-build.yaml

```diff
@@ -13,7 +13,7 @@ on:
         description: "Everest operator branch with the feature to checkout"
       vs_branch:
         required: false
-        default: main
+        default: production
         description: "Version Service branch with the feature to checkout"
```

Comment:
Using `production` instead of `main`, because `main` can contain RC builds which means we would always need to install the latest RC to be able to upgrade to FB using CLI. 
I think the most frequent use case for FB is to test upgrade from the latest released version to FB. If we need to include RCs we can always change the `vs_branch` to `main` when running the pipeline